### PR TITLE
Fix some workshop example typos and more explanation for TLS support 

### PR DESCRIPTION
**Pull request by mfelsche**

### CODE REVIEW

File: docs/artefacts/offramps.md [hunk 1]
```diff
@@ -1461,11 +1461,9 @@ Supported configuration options are:
 - `is_non_blocking` - Is the socket configured as non-blocking ( default: false )
 - `ttl` - Set the socket's time-to-live ( default: 64 )
 - `is_no_delay` - Set the socket's Nagle ( delay ) algorithm to off ( default: true )
-- `tls` - The tls config or a boolean value to indicate the use of TLS stream ( default: TCP stream )
-    - `cafile` - The CA (PEM) file (load the default webpki-roots if no file is provided) 
-    - `domain` - The domain to connect to (Takes the host parameter if not provided)
-
-Example:
+- `tls` - If set to `true` or a detailed TLS config with the keys below, will wrap the TCP socket with a TLS session. ( default: `false` )
+    - `cafile` - If provided, only server certificates signed by the Certificate Authority represented by the root certificate in `cafile` are accepted. If not provided, all root certificated distributed by Mozilla via [webpki-roots](https://github.com/rustls/webpki-roots) are loaded).
```
Comment (Licenser) [inline docs/artefacts/offramps.md:1465]: Based on the example below I'm a bit confused if this is the CA file to validate against or the client certificate to provide to the server?

The description here makes me think it's for validating the server certificate. But the example below:

```
        cafile: "path/to/self_signed_certificate.pem"
```

looks like a client cert

Comment (mfelsche) [reply docs/artefacts/offramps.md:1465]: The `cafile` config key stands for certificate authority, so i hope this makes stuff clear. 

In the paragraph below i mentioned that using a custom `cafile` is useful for testing with self-signed certificates, that is why I added this in the config. I hope this makes the example a bit clearer. 

Any suggestion about how to clear things up?

Comment (Licenser) [reply docs/artefacts/offramps.md:1465]: Thinking aobut what would have helped me is probably mentioning that we don't support client certificates and perhaps renaming `path/to/self_signed_certificate.pem` to `path/to/custom_ca.pem`. I think what threw me off is that `path/to/self_signed_certificate.pem` sounds like it being a client certificate not a CA

**Review by Licenser — APPROVED** [review]
<3 confusion cleared up! wooohhh :D :raccoon:

**Pull request merged**